src/app/cold-storage/rebalance-to-cold-wallet.ts

```diff
@@ -81,19 +81,21 @@ export const rebalanceToColdWallet = async (): Promise<boolean | ApplicationErro
   })
   if (rebalanceBtcAmount instanceof Error) return rebalanceBtcAmount
   const amountDisplayCurrencyAmount = displayPriceRatio.convertFromBtc(rebalanceBtcAmount)
-  const amountDisplayCurrency = usdMinorToMajorUnit(
-    amountDisplayCurrencyAmount.amount,
-  ) as DisplayCurrencyBaseAmount
+  const amountDisplayCurrency = minorToMajorUnit({
```

Comment:
I'm noticing here now that there's some cleanup I need to do with calling WalletPriceRatio/DisplayPriceRatio still. I'll probably start building it on these PRs